apps/frontend/src/api/util.ts

```diff
@@ -43,3 +47,26 @@ export const useFetchOptions = ({
     },
   };
 };
+
+export interface BasicOptions {
+  searchQuery?: string;
+  filters?: string[];
+}
+
+export const useApiGet = <T>(
+  endpoint: string,
+  parameters: Record<string, string | string[] | undefined> = {},
+  options?: Parameters<typeof useFetchOptions>[0],
+) => {
+  const url = new URL(`${API_BASE_URL}${endpoint}`);
+  Object.entries(parameters).forEach(([key, value]) => {
+    if (Array.isArray(value)) {
+      value.forEach((item) => url.searchParams.append(key, item));
+    } else if (value) {
```

Comment:
```suggestion
    } else if (value !== undefined) {
```
should we ever want `?x=` :D

Reply:
Or wait, this behavior seems to be used by the components atm. This util should probably still not erase empty string values, seems cleaner if the components just don't pass empty string values in the first place